Demo/Demo/FeatureViewControllers/PayPalDemoViewController.swift

```diff
@@ -6,22 +6,23 @@ class PayPalDemoViewController: FeatureBaseViewController {
 
     // MARK: - UI Components
 
-    lazy var payPalButton: CustomButton = {
-        let payPalButton = CustomButton(title: "PayPal")
-        payPalButton.addTarget(self, action: #selector(didTapPayPalButton), for: .touchUpInside)
-        payPalButton.layer.cornerRadius = 8
-        payPalButton.backgroundColor = .systemBlue
-        payPalButton.tintColor = .white
-        payPalButton.isEnabled = false
-        return payPalButton
-    }()
+    let payPalButton = PayPalButton()
+
+    let payPalCreditButton = PayPalCreditButton()
```

Comment:
maybe we can remove the credit button from the demo for now, as it doesn't link to the credit flow? :smile:

Reply:
I think it may be helpful to showcase the different buttons we offer, at least for now. As merchants will have full control over the flow, they also don't necessarily _have_ to launch the credit flow (though that's obviously what we'd likely want them to do).